### PR TITLE
Remove unused parameter

### PR DESCRIPTION
**Pull request by jianglingxia**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: test/e2e/storage/persistent_volumes.go [hunk 1]
```diff
@@ -220,8 +220,8 @@ var _ = framework.KubeDescribe("PersistentVolumes", func() {
 		Context("with multiple PVs and PVCs all in same ns", func() {
 
 			// define the maximum number of PVs and PVCs supported by these tests
-			const maxNumPVs = 10
-			const maxNumPVCs = 10
+			//const maxNumPVs = 10
```
Comment (jsafrane) [inline test/e2e/storage/persistent_volumes.go]: Well, if the constants are really unused then please remove them instead of commenting out.

Comment (jianglingxia) [inline test/e2e/storage/persistent_volumes.go]: thanks  for your advice ,I have updated it !

File: test/e2e/storage/persistent_volumes.go [hunk 2]
```diff
@@ -220,8 +220,6 @@ var _ = framework.KubeDescribe("PersistentVolumes", func() {
 		Context("with multiple PVs and PVCs all in same ns", func() {
 
 			// define the maximum number of PVs and PVCs supported by these tests
```
Comment (jsafrane) [inline test/e2e/storage/persistent_volumes.go]: this comment becomes useless now, please remove it

**Comment by jsafrane** [comment]
removed lgtm, @jianglingxia please squash the commits into one, we don't want to have review updates in our git repository, thanks!

**Comment by jianglingxia** [comment]
ok,I have squashed it into one commit ,thanks!

**Pull request merged**